packages/gatsby/src/schema/__tests__/queries.js

```diff
@@ -45,7 +45,7 @@ describe(`Query schema`, () => {
       } else if (api === `createResolvers`) {
         return [
           args[0].createResolvers({
-            Frontmatter: {
+            MarkdownFrontmatter: {
```

Comment:
Does the custom name for nested fields still work as well?

Reply:
Yeah, that is artifact from old test changes when it didn't. thanks